src/components/PWAInstructions/index.tsx

```diff
@@ -118,12 +119,21 @@ const devices = {
   },
 };
 
-const PWAInstructions = (props) => {
-  const [device, setDevice] = useState(null);
-  const [platform, setPlatform] = useState(null);
+type props = {
+  open: boolean;
+  handleDisplay: () => void;
+  deferredPWAPrompt: typeof BeforeInstallPromptEvent;
```

Comment:
This is not actually the correct `BeforeInstallPromptEvent`. Apparently, that event is non-standard, only supported in Chrome, and so the type is not officially available anywhere. Here is a good guide on how to add it properly: https://stackoverflow.com/a/67171375